components/scream/src/share/scream_pack_kokkos.hpp

```diff
@@ -278,17 +282,31 @@ void host_to_device(const Kokkos::Array<typename ViewT::value_type::scalar const
   }
 }
 
+// Sugar for when size is uniform
+template <size_t N, typename ViewT>
+void host_to_device(const Kokkos::Array<typename ViewT::value_type::scalar const*, N>& data,
```

Comment:
Are the original host_to_device and opposite unit tested? If not, please do. In either case, add tests for these convenience functions, too.

Reply:
Yes, they are unit tested. I will add tests for convenience functions.